app/models/framework_response/collection.rb

```diff
@@ -0,0 +1,51 @@
+class FrameworkResponse
+  class Collection < FrameworkResponse
+    validates :value_text, absence: true
+    validate :validate_presence, on: :update, if: -> { framework_question.required && parent.nil? }
```

Comment:
```suggestion
    validate :validate_value_presence, on: :update, if: -> { framework_question.required && parent.nil? }
```

Reply:
moved this validation to parent class as it was repeated everywhere